testing/tests/DevExpress.ui.widgets/accordion.tests.js

```diff
@@ -566,6 +566,85 @@ QUnit.module('widget options', moduleSetup, () => {
         assert.equal($element.find('.' + ACCORDION_ITEM_CLASS).eq(1).outerHeight(), (widgetHeight - closedItemHeight * closedItemsCount) / openedItemsCount, 'opened item content height is correct');
         assert.equal($element.find('.' + ACCORDION_WRAPPER_CLASS).height(), widgetHeight, 'item container height is correct');
     });
+
+    QUnit.test('closed items should have correct height if async template is used (T1166943)', function(assert) {
+        const $element = $('#html-template-accordion');
+        const clock = sinon.useFakeTimers();
+        const items = [
+            { ID: 1 },
+            { ID: 2 },
+            { ID: 3 },
+            { ID: 4 }
+        ];
+        $element.dxAccordion({
+            dataSource: items,
+            itemTitleTemplate: 'custom',
+            templatesRenderAsynchronously: true,
+            integrationOptions: {
+                templates: {
+                    custom: {
+                        render: function({ container, onRendered }) {
+                            setTimeout(() => {
+                                $('<div>Test1</div>').appendTo(container);
+                                onRendered();
+                            }, 10);
+                        }
+                    }
+                }
+            }
+        });
+
+        clock.tick(50);
+
+        const closedItems = $element.find(`.${ACCORDION_ITEM_CLOSED_CLASS}`);
+
+        assert.strictEqual(closedItems.length, 3);
+
+        for(let i = 0; i < closedItems.length; i++) {
+            assert.strictEqual(closedItems.eq(i).outerHeight(), 42.4219);
```

Comment:
Can we make this assert more common? Using of a such strict assertion value can make this test unstable / red if browser work changes a bit or we change a design

Reply:
As a minimum, we can use `roughEqual` with `1` px delta